tests/Integration/Scenario/traces/get-common-chats.json

```diff
@@ -872,6 +872,100 @@
                     "bot_inline_placeholder": null,
                     "lang_code": null
                 },
+                "users": [{
```

Comment:
Traces SHOULD NOT be altered manually!

https://github.com/Postuf/telegram-osint-lib/blob/2c12e36b8ddaa6bda29a09fea33dfd112104b519/src/MTSerialization/OwnImplementation/maps/layer_140.json#L17604